en/includes/guides/organization-management/organization-applications.md

```diff
@@ -54,37 +55,112 @@ and allow access to the API Resources which belongs to the particular organizati
 created with the below conditions.
 
 - Application's protocol is OAuth2.
-- Only client credentials, password and refresh grant types can be used.
+- Only authorization code, client credentials, password and refresh grant types can be used.
+- Only Standard-Based Application and M2M Application templates can be used.
 
-Organization application can be created by using the [Organization application Add]({{base_path}}/apis/organization-apis/organization-application-mgt/#tag/Applications/operation/createApplication) API request. A bearer token with the `internal_org_application_mgt_create` scope, which was issued for the corresponding organization will be needed to invoke this API.
+Organization applications can be created by using the {{ product_name }} Console. You can switch to the organization from the Organization tab and in the organization's console, go to the Applications tab and you can create OAuth2 applications.
+
+![Organization Application Create]({{base_path}}/assets/img/guides/applications/organization-applications/organization-application-create.png){: style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```

Comment:
This will be a broken image for IS-7.2.0 and 7.1.0 docs spaces, show conditionally